var/spack/repos/builtin/packages/simmetrix-simmodsuite/package.py

```diff
@@ -188,6 +226,7 @@ class SimmetrixSimmodsuite(Package):
     variant('discrete', default=False, description='enable discrete modeling')
     variant('acis', default=False, description='enable acis modeling')
     variant('parasolid', default=False, description='enable parasolid modeling')
+    variant('opencascade', default=False, description='enable opencascade modeling')
```

Comment:
I think here you might be missing a `when=` argument:
```suggestion
    variant('opencascade', default=False, when='@16.0-220312', description='enable opencascade modeling')
```

Reply:
@alalazo Good catch.  Thank you.  I committed your suggestion and extended the version range to include future versions https://github.com/spack/spack/pull/29759/commits/484307bea3f6b38ac3e6850fbb8de881c036f2a1.